PC02_Graffiti_EPPERSON.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+'''
+Turtle starter code
+ATLS 1300
+Author: Dr. Z
+Author: Max Epperson
+May 29, 2020
+'''
+
+from turtle import * #import the library of commands that you'd like to use
+
+colormode(255)
+
+# Create a panel to draw on. 
+panel = Screen()
+w = 750 # width of panel
+h = 750 # height of panel
+panel.setup(width=w, height=h) #600 x 600 is a decent size to work on. 
+#You can experiment by making it the size of your screen or super tiny!
+
+# Create a colorful background and add Bezos image to it
+image = "Bezos.gif"
+panel.bgcolor("lightsteelblue1")
+panel.bgpic(image)
+
+#=======Add your code here======
+home()
+showturtle()
+up()
+clear()
+goto(120,-90)
+clear()
+
+color(168,204,215) #piss jar base
+width(5)
+down()
+fillcolor(225,225,20)
+begin_fill()
+left(90)
+forward(50)
+right(45)
+goto(130,-30)
+right(45)
+forward(100)
+right(45)
+goto(240,-40)
+right(45)
+forward(150)
+right(45)
+goto(230, -200)
+right(45)
+forward(100)
+right(45)
+goto(120,-190)
+right(45)
+forward(100)
+end_fill()
+
+up()            #glass details
+goto(130, -190)
+down()
+right(90)
+forward(100)
+up()
+goto(130,-40)
+down()
+forward(100)
+
+up()            #jar lid base
+goto(130,-25)
+color("black")
+fillcolor("black")
+down()
+begin_fill()
+forward(100)
+left(90)
+forward(20)
+left(90)
+forward(100)
+left(90)
+forward(20)
+end_fill()
+ 
+up()           #jar lid lines
+goto(135,-6)
+color("white")
+width(2)
+down()
+forward(19)
+up()
+goto(145,-6)
+down()
+forward(19)
+up()
+goto(155,-6)
+down()
+forward(19)
+up()
+goto(165,-6)
+down()
+forward(19)
+up()
+goto(175,-6)
+down()
+forward(19)
+up()
+goto(185,-6)
+down()
+forward(19)
+up()
+goto(195,-6)
+down()
+forward(19)
+up()
+goto(205,-6)
+down()
+forward(19)
+up()
+goto(215,-6)
+down()
+forward(19)
+up()
+goto(225,-6)
+down()
+forward(19)
+
+up()           #piss bubbles
+goto(150,-70)
+color(199,193,22)
+down()
+dot(20)
+up()
+goto(170,-90)
+dot(10)
+goto(150,-100)
+dot(5)
+
```

Comment:
Add dot notation for platter and soft code recurring values
```suggestion
slategrey = (190,194,203) 
platYcor = -205
#draws a slate grey platter and gives it a black outline
platTurt.up()             
platTurt.goto(50,platYcor)
platTurt.width(5)
platTurt.color("black")
platTurt.fillcolor(slategrey)
platTurt.down()
platTurt.begin_fill()
platTurt.goto(310,platYcor)
platTurt.goto(240,platYcor-15)
platTurt.goto(120,platYcor-15)
platTurt.goto(50,platYcor)
platTurt.end_fill()
```